orion-ui/src/pages/FlowRunCreate.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <p-layout-default v-if="deployment">
+    <template #header>
+      <PageHeadingFlowRunCreate :deployment="deployment" />
+    </template>
+
+    <FlowRunCreateForm :deployment="deployment" @submit="createFlowRun" @cancel="goBack" />
+  </p-layout-default>
+</template>
+
+<script lang="ts" setup>
+  import { FlowRunCreateForm, PageHeadingFlowRunCreate, DeploymentFlowRunCreate, ToastFlowRunCreate, inject, flowRunRouteKey } from '@prefecthq/orion-design'
+  import { showToast } from '@prefecthq/prefect-design'
+  import { useSubscription, useRouteParam } from '@prefecthq/vue-compositions'
+  import { computed, h } from 'vue'
+  import { useRouter } from 'vue-router'
+  import { routes } from '@/router'
+  import { deploymentsApi } from '@/services/deploymentsApi'
+
+  const deploymentId = useRouteParam('id')
+  const router = useRouter()
+  const flowRunRoute = inject(flowRunRouteKey)
+
+  const deploymentSubscription = useSubscription(deploymentsApi.getDeployment, [deploymentId])
+  const deployment = computed(() => deploymentSubscription.response)
+
+  const createFlowRun = async (deploymentFlowRun: DeploymentFlowRunCreate): Promise<void> => {
+    try {
+      const flowRun = await deploymentsApi.createDeploymentFlowRun(deploymentId.value, deploymentFlowRun)
+      const toastMessage = h(ToastFlowRunCreate, { flowRun, flowRunRoute, router, immediate: !!deploymentFlowRun.state?.stateDetails?.scheduledTime })
+      showToast(toastMessage, 'success')
+      router.push(routes.deployment(deploymentId.value))
+    } catch (error) {
+      showToast('Something went wrong trying to create a flow run', 'error')
```

Comment:
Action item for later but this message should probably go in the localization in orion-design.